src/lib/windows-helpers.ts

```diff
@@ -0,0 +1,96 @@
+import { app, dialog, shell } from 'electron';
+import path from 'path';
+import { __ } from '@wordpress/i18n';
+import sudo from 'sudo-prompt';
+import { STUDIO_DOCS_WINDOWS_SPEED_UP_SITES_URL } from '../constants';
+import { loadUserData, saveUserData } from '../storage/user-data';
+
+export async function promptWindowsSpeedUpSites( {
+	skipIfAlreadyPrompted,
+}: {
+	skipIfAlreadyPrompted: boolean;
+} ) {
+	const userData = await loadUserData();
+
+	if (
+		process.platform !== 'win32' ||
+		( skipIfAlreadyPrompted && typeof userData.promptWindowsSpeedUpResult !== 'undefined' )
+	) {
+		return;
+	}
+
+	const NOT_INTERESTED = __( "I'm not interested." );
+	const MANUAL_UPDATE = __( "I'll do it my own by following the documentation." );
+	const AUTOMATIC_UPDATE = __( 'Sounds good, do it for me.' );
+
+	const buttons = [ NOT_INTERESTED, MANUAL_UPDATE, AUTOMATIC_UPDATE ];
+
+	const { response } = await dialog.showMessageBox( {
+		type: 'question',
+		buttons,
+		title: __( 'Want to speed up sites?' ),
+		message: __(
+			"If the Real-Time Protection Service of Windows Defender is enabled on your machine, it may slow down the process of creating and starting a site.\n\nTo enhance site speed, it's recommended adjusting the configuration accordingly.\n\nThe app can do this automatically for you, or alternatively, you can follow the documentation."
```

Comment:
I'd appreciate any feedback on the messaging. The content is mostly based on [the Documentation](https://developer.wordpress.com/docs/developer-tools/studio/#15-how-can-i-make-studio-faster-on-windows-).

Reply:
Could we maybe add the `Needs Copy Review` label or ping `@\Automattic/editorial` to get some extra help from the Publications team? I don't feel overly confident in my copy abilities, but feel the `adjusting the configuration accordingly` line is perhaps a bit too vague in this context. 

Maybe it could be updated to something like the following?

`For optimal performance, we recommend disabling this service for the Studio app.`

Reply:
> Could we maybe add the `Needs Copy Review` label or ping `@\Automattic/editorial` to get some extra help from the Publications team? I don't feel overly confident in my copy abilities, but feel the `adjusting the configuration accordingly` line is perhaps a bit too vague in this context.

Good idea, I've just added the label.

> Maybe it could be updated to something like the following?
> 
> `For optimal performance, we recommend disabling this service for the Studio app.`

The idea is to exclude the app from this service, not disabling as Windows Defender is meant to protect user's machine from security threats.

Reply:
Oh, that's what I was trying to convey with the `for the Studio app` part of the suggestion, but see it wasn't clear. 😅 How about something like the following?

`For optimal performance, we recommend excluding the Studio app from this service.`

I'm sure the Publications team will have some better suggestions than myself, though!

Reply:
> Oh, that's what I was trying to convey with the `for the Studio app` part of the suggestion, but see it wasn't clear. 😅

Ah, true. I've just re-read it and I found it clearer now 😅 .

> [...] How about something like the following?
> `For optimal performance, we recommend excluding the Studio app from this service.`

Sounds good, I'll update the message.

Reply:
I applied this suggestion in https://github.com/Automattic/studio/pull/211/commits/df02d636a21ccbc296464856772f324ee5fb6b3b.